frontend/src/components/viewers/MarkdownViewer.tsx

```diff
@@ -53,13 +54,16 @@ position: relative;
 }
 `);
 
+const MAX_MARKDOWN_STR_LENGTH = 50 * 1000 * 8; // 50KB
+
 export interface MarkdownViewerConfig extends ViewerConfig {
   markdownContent: string;
 }
 
-interface MarkdownViewerProps {
+export interface MarkdownViewerProps {
   configs: MarkdownViewerConfig[];
   maxDimension?: number;
+  maxMarkdownStrLength?: number;
```

Comment:
nit: the component already tells this is for markdown, this field can be simpler just named e.g. `maxLength`

Reply:
Done.